examples/volume_mesher_v2_from_inputs.py

```diff
@@ -1,9 +1,10 @@
-import os
-
-os.environ["FLOW360_BETA_FEATURES"] = "1"
+# Comment out to pass linter. Uncomment below to run this example
+# import os
+# os.environ["FLOW360_BETA_FEATURES"] = "1"
```

Comment:
I really cannot come up with a good way of fixing this without introducing a new error... Ruff complains that the imports are not at root level for this one.

Reply:
Beta_features are somewhat hacky right now and I think they aren't used anywhere - in the refactor we will probably need to come up with a better solution for feature flagging - I think this could be removed for now? @maciej-flexcompute

Reply:
I think @feilin-flexcompute added this BETA tag for new volume mesher pipeline so it is used for that only.